packages/generator-langium/templates/packages/extension/package.json

```
@@ -1,4 +1,7 @@
 {
+    "name": "<%= extension-name %>-extension",
```

Minor: This name will eventually end up as the ID for the extension on the vscode marketplace (assuming the adopter uploads it there). We should align to other extensions there and rather call it `vscode-whatever`, like [`vscode-yaml`](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml).
```suggestion
    "name": "vscode-<%= extension-name %>",
```

---

Will do

---

Done ⬇️